mozartdata/transforms/fact/inventory.sql

```diff
@@ -0,0 +1,25 @@
+-- CREATE OR REPLACE TABLE fact.inventory
+--             COPY GRANTS  as
+SELECT sku,
+	   display_name,
+	   LOCATION_NAME,
+	   FIVETRAN_SNAPSHOT_DATE_PST as snapshot_date,
+	   total_quantity_on_hand as quantity, --using this one because its the highest level one available for unioning purposes
+	   'Netsuite' AS source
+FROM fact.NETSUITE_INVENTORY_LOCATION
+UNION ALL
+SELECT sku,
+	   display_name,
+	   store,
```

Comment:
it doesn't really matter, but change this
 ```suggestion
	   store as location_name,
```